custom-script/machine-learning-evalscript.ipynb

```diff
@@ -1290,31 +1171,29 @@
     "You can check it in `utils.py` file."
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": 31,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "sh_prediction = predict_on_sh(js_model, eop.bbox, (64, 64), eop.timestamp[0], config)"
-   ]
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's run the predictions with the model we've trained, using data from the `eop`:"
+    "Let's make a function useing it and run the predictions with the model we've trained, using data from an `EOPatch` (for later comparison)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 32,
+   "execution_count": 27,
    "metadata": {},
    "outputs": [],
    "source": [
-    "features = eop.data[\"FEATURES\"][0]\n",
-    "height, width, nfeats = features.shape\n",
-    "model_prediction = model.predict_proba(features.reshape(height * width, nfeats))[..., -1].reshape(height, width)"
+    "def get_predictions(patch):\n",
```

Comment:
Perhaps this should also be moved to utils?

Reply:
It could be in that case it is possible to keep notebook "dry" and show example what 92% prediction mean on eop image i would put "error plot" after print (  F1, Recall, Precision) ...

Reply:
done